plugins/WPML.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * ACF Pro Plugin.
+ *
+ * @package Junaidbhura\Composer\WPProPlugins\Plugins
+ */
+
+namespace Junaidbhura\Composer\WPProPlugins\Plugins;
+
+/**
+ * AcfPro class.
+ */
+class WPML {
+
+    /**
+     * The version number of the plugin to download.
+     *
+     * @var string Version number.
+     */
+    protected $version = '';
+
+    /**
+     * The slug of which plugin to download.
+     *
+     * @var string Plugin slug.
+     */
+    protected $slug = '';
+
+    /**
+     * WpAiPro constructor.
+     *
+     * @param string $version
+     * @param string $slug
+     */
```

Comment:
Change method block comment to reflect WPML plugin.

```suggestion
    /**
     * Wpml constructor.
     *
     * @param string $version
     * @param string $slug
     */
```